assessments/guide/joint-evaluation.md

```diff
@@ -0,0 +1,312 @@
+[self-assessment](self-assessment.md) provide the community with the SIG-Security
+Evaluation of the project.  Both of these documents may be used and references as part
+of a security audit.
+
+## Intended Use
+
+* Target Users and Use Cases. Provide a mapping from [standard personas](../../usecases.md)
+to the nomenclature used in your project docs (which you should then use
+consistently for the remainder of this document).  Describe the scenarios in
+which the project is expected to be used. This must be specific enough to
+provide context for analysis. For example:
+
+    Flibble can be used in any cloud environment.  Three diverse examples are as
+follows:
+    1. when a Flibble server is used by legacy servers as a database for salted
+       password hashes.
+    2. a Flibble cloudlet may be run on virtualized fog hardware near smartphone
+       users.
+    3. a Flibble distributed service may serve as a backend for a Notary image
+       registry.)
+
+* Operation.  A description of the operational aspects of the system, such as
+  how keys are likely to be managed and stored.
+
+## Project Design
+
+* Design.  A description of the system design that discusses how it works. This
+  is especially critical for any mechanisms that relate to the security of the
+system.  Include architecture and network (if applicable) information such as
+encryption of traffic between services, access control types (RBAC, etc.) and
+enforcement, or security logging, etc.
+
+* Data flow diagram/Architecture diagram
+
+### Functions and features
+
+* Critical.  A listing with brief description of functions and features that are
+  critical to the project's ability to meet its intended use.  It is recommended
+these be covered in the threat model.
+* Relevant.  A listing with brief description of the functions and
+  features of the project that perform a security relevant function.  It is
+recommended these be covered in the threat model.
+
+#### Security functions and features
+
+This section should be pulled from the self-assessment
+
+## Configuration and Set-Up
+
+* Default.  Documentation describing the default configuration of the project
+  with initial set-up instructions (link to docs is acceptable). Documentation
+should identify potential security risks/trade-offs of the default config.
+* Secure. Documentation describing recommended secure configuration and set-up
+  instructions, beyond defaults with justification for selection and trade-offs
+(link to docs is acceptable).
+* Advanced Secure. If applicable, documentation describing advanced settings for
+  most hardened configuration of the project to include justification for
+selection and trade-offs (link to docs is acceptable).
+
+## Project Compliance
+
+This can be pulled from the self-assessment
+* Compliance.  List any security standards or sub-sections the project is
+  already documented as meeting (PCI-DSS, COBIT, ISO, GDPR, etc.).
```

Comment:
I'm not sure if these standards are what we should be asking about. These mostly relate to products or deployments, which may not be applicable to most of the evaluations we do. I'm thinking that based on what we are trying to find out here: Can I use this for compliance, or what is the current evaluated security posture of the project?

For the first, we can ask for compliance mapping mainly targeted at NIST 800-53, or HiTrust. And for the later, examples of use of projects in compliant environments or security audits.